update all packages, add example

epfromer opened the pull request:
I updated all packages and stuck with axios since this repo doesn't necessarily work within DOM and have access to DOM fetch().  I also added an example app which I used to test the library.  All original tests pass.

epfromer committed:
update all packages, add example

adumont commented:
Please could you not remove axios-curlirize? It's a useful feature to be able to do:

export CURLIRIZE=1
node app.js

and see all the requests shown as Curl commands. (For debugging purposes for example).

epfromer committed:
Readd axios-curlirize

epfromer commented:
Apologies, I removed that because I had problems with modules, and neglected to put it back.  I have now checked in a version with it restored.  It is hardwired to v1.3.7 because that uses CommonJS modules, whereas v2.0.0 moves to ES Native Modules.  I also removed node-fetch from the package.json, which was left over from exploration of using that instead of axios.

milanmdev commented:
Any updates on the status of this PR?
